ansible/roles/machine/provision_webui/templates/ui_test.conf.j2

```diff
@@ -0,0 +1,31 @@
+ipa_admin: admin
+ipa_password: Secret.123
+
+{% for host in groups['all'] %}
+{% if host.startswith('master') %}
+ipa_server: {{ hostvars[host]['inventory_hostname'] }}.ipa.test
+ipa_ip: {{ hostvars[host]['ansible_default_ipv4']['address'] }}
+ipa_domain: ipa.test
+ipa_realm: IPA.TEST
+{% endif %}
+{% endfor %}
+
+# Uncomment when IPA is installed without CA
+#no_ca: True
+
+# Uncomment when IPA is installed without DNS server
+#no_dns: True
+
```

Comment:
Could you please add configuration option for Geckodriver.log ? 

```
# Geckodriver setup:
# =================
# log file has to be somewhere, where user has rights to write into file
geckodriver_log_path: /home/me/.ipa/geckodriver.log
```
and set it as PR-CI needs. We might need to collect this log in the future (or is it possible collect it right now?)

Reply:
Why, yes -- we can!

![very log](https://i.imgflip.com/210do7.jpg)